ocs_ci/utility/kms.py

```diff
@@ -73,6 +73,7 @@ def __init__(self):
         self.cluster_id = None
         # Name of kubernetes resources
         # for ca_cert, client_cert, client_key
+        self.kms_type = constants.VAULT_TOKEN
```

Comment:
self.kms_type sounds like a misnomer, I feel it should be `self.kms_encryption_type` if I understand correctly. feel free to correct me.

Reply:
Changed it to `kms_auth_type`